src/modules/populate.js

```diff
@@ -0,0 +1,26 @@
+import toDoList from './elements.js';
+
+const populate = ({ description }) => {
+  const listItem = document.createElement('li');
+  listItem.classList.add('border', 'list-flex');
+  toDoList.appendChild(listItem);
+
+  const checkbox = document.createElement('input');
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.classList.add('check-list');
+
+  const text = document.createElement('input');
+  text.setAttribute('placeholder', description);
+  text.setAttribute('type', 'text');
+  text.setAttribute('disabled', '');
+
+  const div = document.createElement('div');
+  div.append(checkbox, text);
+
+  const btn = document.createElement('button');
+  btn.setAttribute('type', 'button');
+  btn.innerHTML = '<i class="bi bi-three-dots-vertical"></i>';
+  listItem.append(div, btn);
+};
```

Comment:
- [ ] Kindly make sure that the list is populated in the DOM dynamically in the order of the index property, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement,

Reply:
@ShadyShawkat  I don't understand what you mean because it is already populated in the order of the index

Reply:
Hey @Kamba56, the requirements says to populate the DOM in the order of the index values. that means if you were to have mixed values for the indices, you first would want to sort them according to their index values and then populate the DOM.